Tasks/AndroidSigningV2/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "homepage": "https://github.com/microsoft/vsts-tasks#readme",
   "dependencies": {
+    "@types/node": "^6.0.0",
```

Comment:
6.0.101 for these tasks?

Reply:
I don't think that was any issue, and it should match with the (^) symbol